contrib/runners/orquesta_runner/tests/unit/test_error_handling.py

```diff
@@ -100,19 +100,51 @@ def sort_wf_runtime_errors(self, errors):
         return sorted(errors, key=lambda x: x.get('task_id', None))
 
     def test_fail_inspection(self):
+        expected_errors = [
+            {
+                'type': 'content',
+                'message': 'The action "std.noop" is not registered in the database.',
+                'schema_path': 'properties.tasks.patternProperties.^\w+$.properties.action',
+                'spec_path': 'tasks.task3.action'
+            },
+            {
+                'type': 'context',
+                'language': 'yaql',
+                'expression': '<% ctx().foobar %>',
+                'message': 'Variable "foobar" is referenced before assignment.',
+                'schema_path': 'properties.tasks.patternProperties.^\w+$.properties.input',
+                'spec_path': 'tasks.task1.input',
+            },
+            {
+                'type': 'expression',
+                'language': 'yaql',
+                'expression': '<% <% succeeded() %>',
+                'message': (
+                    'Parse error: unexpected \'<\' at '
```

Comment:
I know this question belongs to Orquesta more than here, but do we also have access to he line number where the error occurred? And can we include that in the error?

I see there is path which can help the user, but line number would make troubleshooting even easier.

Reply:
Line number is definitely nice to have. But there are multiple types of inspection here. As @bigmstone stated, some of these categories (i.e. contents, context, expressions, etc.) are related to content of the workflow definition and not jsonschema or YAML specific. The line numbers need to be determine by ourselves in those cases. Since we already know the path from inspection, we need a function that can map dot notation to line number. I did a brief search but can't identify any library that does this. And I already had this chat with @enykeev. He's ok without line numbers. I think he can already resolve dot notation to line number currently in st2flow.